torchci/components/NavBar.tsx

```diff
@@ -61,10 +61,15 @@ function NavBar() {
               </Link>
             </li>
             <li>
-              <Link prefetch={false} href="https://hud.pytorch.org/metrics">
+              <Link prefetch={false} href="/metrics">
                 Metrics
               </Link>
             </li>
+            <li>
+              <Link prefetch={false} href="/kpis">
```

Comment:
Let's not add this yet till we've finished the page

Reply:
Sure